### PR TITLE
syncthing_exporter: init at 0.3.0

### PR DESCRIPTION
**Pull request by bhankas**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/syncthing_exporter/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  vendorSha256 = "sha256-JbrCgt5Tu65IZNqtz3OvIESwYO1p6cH1ohzkVc1pwTE=";
+
+  postInstallCheck = ''
+    $out/bin/syncthing_exporter --version > /dev/null
+  '';
+
+  meta = with lib; {
+    description = "Syncthing metrics exporter for prometheus";
```
Comment (fabaff) [inline pkgs/applications/misc/syncthing_exporter/default.nix:21]: [Prometheus](https://prometheus.io/) is using `Prometheus` not `prometheus`. We should respect their way of using the name.

```suggestion
    description = "Syncthing metrics exporter for Prometheus";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
This should get a nixos module entry, too.

**Comment by Ma27** [comment]
Also, all prometheus exporters have a common naming scheme in nixpkgs, namely `prometheus-<name>-exporter` for packages.

**Comment by remedialchaos** [comment]
> Also, all prometheus exporters have a common naming scheme in nixpkgs, namely `prometheus-<name>-exporter` for packages.

Thanks! I noticed that after pushing. I made the changes locally but haven't had time to test yet. Will update probably by weekend.

**Comment by bbigras** [comment]
Any progress on this?

**Comment by e3bc54b2** [comment]
Hi, sorry I’ve been pccupied with other aspects of life, and cannot tell when I can get back to fixing the comments. We can abandon this one.

**Pull request closed without merging**